### ISSUE TITLE
cgroup2: failed to create a new cgroup 

### ISSUE THREAD

**Issue by markzhang0928** [containerd/cgroups#152]
when I create a new cgroup by using cgctl, it returns nil pointer dereference as follows: 

```
./cgctl --mountpoint /sys/fs/cgroup/unified new /testb
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6e33b4]

goroutine 1 [running]:
github.com/containerd/cgroups/v2.(*Resources).EnabledControllers(...)
	/root/go/src/github.com/containerd/cgroups/v2/manager.go:105
github.com/containerd/cgroups/v2.NewManager(0x7fffa262777d, 0x16, 0x7fffa2627798, 0x6, 0x0, 0x2, 0x2, 0x3a)
	/root/go/src/github.com/containerd/cgroups/v2/manager.go:180 +0x174
main.glob..func1(0xc0000ae6e0, 0x0, 0xc0000869f0)
	/root/go/src/github.com/containerd/cgroups/cmd/cgctl/main.go:81 +0x1b9
github.com/urfave/cli.HandleAction(0x79a8c0, 0x833cd0, 0xc0000ae6e0, 0xc0000ae6e0, 0x0)
	/root/go/pkg/mod/github.com/urfave/cli@v1.22.2/app.go:523 +0xbe
github.com/urfave/cli.Command.Run(0x817e6f, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, 0x820e2e, 0x13, 0x0, ...)
	/root/go/pkg/mod/github.com/urfave/cli@v1.22.2/command.go:174 +0x51c
github.com/urfave/cli.(*App).Run(0xc0000b4380, 0xc000096000, 0x5, 0x5, 0x0, 0x0)
	/root/go/pkg/mod/github.com/urfave/cli@v1.22.2/app.go:276 +0x725
main.main()
	/root/go/src/github.com/containerd/cgroups/cmd/cgctl/main.go:63 +0x419
```

https://github.com/containerd/cgroups/blob/master/v2/manager.go#L189
https://github.com/containerd/cgroups/blob/master/cmd/cgctl/main.go#L80

**Comment by AkihiroSuda** [comment]
ping @Zyqsempai

**Comment by mikebrow** [comment]
hey.. have a new contributor learning the ropes he was going to try to push a pr for this one seems straight forward.

**Comment by mikebrow** [comment]
@markzhang0928 issue is resolved by #162
